java-client/src/main/java/org/opensearch/client/json/JsonpUtils.java

```diff
@@ -159,6 +159,21 @@ public static <T> void serialize(T value, JsonGenerator generator, @Nullable Jso
         }
     }
 
+    /**
+     * Serializes an object to a JSON string.
+     *
+     * @param value the object to serialize
+     * @param mapper the JSON mapper to use for serialization
+     * @return the JSON string representation
+     */
+    public static String toJsonString(Object value, JsonpMapper mapper) {
```

Comment:
This method is used in test only, let's move it off there instead of adding to `JsonpUtils`'s public API